app/stores/UiStore.ts

```diff
@@ -134,10 +135,6 @@ class UiStore {
         this.tocVisible = newData.tocVisible;
       }
     });
-
-    autorun(() => {
```

Comment:
Removing the autorun means we won't write to storage unless the user _in this tab_ interacts.